### ISSUE TITLE
CVE-2019-20330 not reporting for jackson-databind

### ISSUE THREAD

**Issue by msymons** [OSSIndex/vulns#67]
To facilitate future automation, please use the following format

**Advisory details**
```
  URL:  https://nvd.nist.gov/vuln/detail/CVE-2019-20330
  format: maven
  namespace: com.fasterxml.jackson.core
  name: jackson-databind
  versions: 2.x before 2.9.10.2
```

**More information**
Note that `org.codehaus.jackson:jackson-mapper-asl` and  `org.codehaus.jackson:jackson-mapper-lgpl` are possibly also vulnerable for versions 1.5.0 to 1.9.13

**Comment by msymons** [comment]
@ken-duck, any update?  The lag between CVE publishing and linkage to purls in OSS Index is causing my developers to miss opportunities to remediate simply because they do not know that jackson-databind 2.9.10.1 is vulnerable.

**Comment by ken-duck** [comment]
Many apologies. I will look into this tonight.

**Comment by ken-duck** [comment]
We do have some refactoring under way that will change how the research for these issues occur, and subsequently will eliminate bugs like these. It is a rather large rework, though, so will take rather some time to get done.

**Comment by ken-duck** [comment]
Yup, looks like one of our services had a glitch and has been missing some data. I gave it a kick and you should see the missing data sometime tomorrow. As mentioned, we are in the process of doing a rather major refactoring on the vulnerability research code which will replace these occasionally rickety services. Once complete I expect they will be rock solid, but till then I am coaxing the existing services to keep going.

I will keep my eye on this one for a while to make sure it doesn't glitch again.

Thanks for letting me know (and keeping on me). I am rather overburdened at the moment so things fall off the stack. The OSS Index initiative has actually recently been assigned a couple of new teams to work on it. Once we get over the learning curve and switchover hump I expect things will start to run more smoothly.

**Comment by ken-duck** [comment]
I'm trying to figure out the best way to implicate `org.codehaus.jackson:jackson-mapper-asl` and `org.codehaus.jackson:jackson-mapper-lgpl`. They are a bit more complicated (based on how OSS Index works), but `jackson-databind` will work.

**Comment by ken-duck** [comment]
I verified that the main package is now vulnerable.
